### PR TITLE
Shantanu HDP CHanges

### PR DESCRIPTION
**Pull request by Nishant-K-Raj**
Adding HDP Changes

### CODE REVIEW

File: upgrade-toolkit/HDP Cluster Inventory/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+> 10. System Python Version
+
+#### The Cloudera Version Information Sheet will gather the following information:
+> 1. HDP Version
+> 2. Cluster Name
+> 3. Backend DB Type
+
+
+## Requirements: 
+> 1. AMBARI user and password must have Read privileges on Cloudera Manager
```
Comment (Chaffelson) [inline upgrade-toolkit/HDP Cluster Inventory/README.md]: Probably not on Cloudera Manager

File: upgrade-toolkit/HDP Cluster Inventory/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Inventory CSV Generator
```
Comment (Chaffelson) [inline upgrade-toolkit/HDP Cluster Inventory/README.md]: The directory name should not have spaces

File: upgrade-toolkit/HDP Cluster Inventory/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Requirements: 
+> 1. AMBARI user and password must have Read privileges on Cloudera Manager
+> 2. The script must be run from a node that has password-less ssh acess to all hosts in the cluster
+> 3. python3 must be installed
+> 4. Import the following packages
+> 5. Ambari_domain is inout to python script
+
+```sh
+import subprocess
```
Comment (Chaffelson) [inline upgrade-toolkit/HDP Cluster Inventory/README.md]: The python requirements should in a requirements.txt file

**Review by Chaffelson — CHANGES_REQUESTED** [review]

File: upgrade-toolkit/HDP Cluster Inventory/README.md (hunk 1 above)
Comment (shantanugope) [reply upgrade-toolkit/HDP Cluster Inventory/README.md]: changed to Amabri.. it was typo

File: upgrade-toolkit/HDP Cluster Inventory/README.md (hunk 3 above)
Comment (shantanugope) [reply upgrade-toolkit/HDP Cluster Inventory/README.md]: created a new file for requirements

File: upgrade-toolkit/HDP Cluster Inventory/README.md (hunk 2 above)
Comment (shantanugope) [reply upgrade-toolkit/HDP Cluster Inventory/README.md]: removed spaces

**Review by Chaffelson — APPROVED** [review]

**Pull request merged**